project1/client/client_tasks/download_task.h

```diff
@@ -8,27 +12,32 @@
 namespace client_tasks {
 class DownloadTask : public thread_pool::Task {
  public:
-  DownloadTask(const std::string& filename, std::vector<uint8_t> &incoming_buf,
-               size_t buf_size, int client_fd,
-               wire_protocol::MessageParser<google::protobuf::Message> &parser
-               ) {
+  DownloadTask(const std::string& filename, size_t buf_size, int client_fd) {
     filename_ = filename;
-    incoming_file_buf_ = incoming_buf;
     buffer_size_ = buf_size;
     client_fd_ = client_fd;
-    parser_ = parser;
   }
 
   Status RunAtomic() override;
 
   void CleanUp() override;
 
  protected:
+
+  // the name of the file to be retreived
```

Comment:
(minor) Single-line documentation comments should have triple slashes. This is so they will be interpreted as such if we ever decide to run Doxygen. (admittedly unlikely)